android/app/src/main/res/layout/large_forecast_widget_layout.xml

```diff
@@ -258,17 +72,6 @@
             tools:text="Viranomaistiedote: 13.11.2024 klo: 11.00: Alueella Lauttasaaren ranni..."
             tools:visibility="visible" />
 
-        <TextView
-            android:id="@+id/updateTimeTextView"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:layout_gravity="center_horizontal"
-            android:textColor="@android:color/black"
-            android:textSize="10sp"
-            android:visibility="visible"
-            tools:ignore="SmallSp"
-            tools:text="11.34.24" />
-
```

Comment:
Can you put this back please?
I would like to see for testing purposes when the widget gets updated.
It can be set to GONE later.